maths/greater_common_divisor.py

```diff
@@ -1,25 +1,25 @@
-"""
-Greater Common Divisor.
+# Python program to find the G.C.D of two input number
 
-Wikipedia reference: https://en.wikipedia.org/wiki/Greatest_common_divisor
-"""
+# define a function
+def computeGCD(x, y):
 
+# choose the smaller number
+    if x > y:
+        smaller = y
+    else:
+        smaller = x
```

Comment:
__smaller = min(x, y)__

Reply:
as I mentioned above.I like to keep codes beginner friendly.